### ISSUE TITLE
Bug With Linking To Warnings in Nested Files

### ISSUE THREAD

**Issue by sh0rtcircuit** [James-Yu/LaTeX-Workshop#245]
### Description

When I have a nested structure, i.e.,
/mainfolder/
that includes my main file "main.tex".

and a subfolder 

/mainfolder/sections
that includes a file "section.tex".

This folder in turn contains a folder for subsections
/mainfolder/sections/subsections
that includes a file "subsection.tex".

The file "subsections.tex" includes a waring (i.e., invalid reference).

When I click on the warning in VSCode (that usually takes me to the location of the warning),

![image](https://user-images.githubusercontent.com/24651737/29528068-f29bee6e-869a-11e7-98fd-f2ef36376491.png)

I get an error that the file could not be found (see attached screenshot):

![image](https://user-images.githubusercontent.com/24651737/29527969-a9dbc06e-869a-11e7-8a5e-7af086789236.png)

Basically the program searches in the wrong folder for the document (sorry for german text).

I attached a zip file with a latex project that illustrates the problem (problem.zip).
[problem.zip](https://github.com/James-Yu/LaTeX-Workshop/files/1239378/problem.zip)

Maybe this helps to fix the problem.

Best regards, André
